policyengine_us/variables/gov/states/ny/nysdps/ny_clean_heat_incentive.py

```diff
@@ -0,0 +1,74 @@
+from policyengine_us.model_api import *
+
+class ny_clean_heat_incentive(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New York State Clean Heat incentive (con Edison)"
+    documentation = "The incentive for purchasing and installing a heat pump"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://cleanheat.ny.gov/assets/pdf/CECONY%20Clean%20Heat%20Program%20Manual%206%203%2024.pdf#page=12"
+    defined_for = StateCode.NY
+
+    def formula(tax_unit, period, parameters):
+        '''
+        Calculates the incentive and the cap for the NYS Clean Heat program.
+
+        Parameters:
+        - parameters (instance)
+        - period (int)
+        - tax_unit (instance)
+
+        Returns:
+        - float: a capped incentive (float). 
+        '''
+        
+        p = parameters(period).gov.states.ny.nysdps.con_edison_clean_heat
+        project_cost = tax_unit("ny_clean_heat_project_cost", period)
+
+        family_type = tax_unit("ny_clean_heat_family_type_category", period) #var0
+        heat_pump = tax_unit("ny_clean_heat_heat_pump_category", period) #var1
+
+        # Residential
+        if family_type == 'RESIDENTIAL':
```

Comment:
I want to use enum values for conditional branching, but they are not recognized in the test. 
I tried the following but nothing worked:
`
        if family_type == ‘Residential:
        if family_type.status == 'RESIDENTIAL':
        if family_type.value == 'RESIDENTIAL':
`
How should I write here?